services/common/src/components/reports/ReportDetailsForm.tsx

```diff
@@ -42,6 +88,40 @@ const ReportDetailsForm: FC<ReportDetailsFormProps> = ({
   const formValues: IMineReport =
     useSelector((state) => getFormValues(FORM.VIEW_EDIT_REPORT)(state)) ?? {};
   const { mine_report_category = "", mine_report_definition_guid = "" } = formValues;
+  const [mineManager, setMineManager] = useState<IParty>();
+  const [mineManagerGuid, setMineManagerGuid] = useState<string>("");
+
+  const partyRelationships: IPartyAppt[] = useSelector((state) => getPartyRelationships(state));
+  const parties = useSelector((state) => getParties(state));
+
+  useEffect(() => {
+    if (!partyRelationships.length) {
```

Comment:
I really dislike how we default this in the reducer to empty array! (instead of null or undefined)
I am wondering if there would be a scenario where the mine just _doesn't have_ any party relationships. Maybe it wouldn't be realistic for them to submit a report? Are all the party relationships in mine_party_appt? If so, `SELECT * FROM mine_report WHERE mine_guid IN
(SELECT mine_guid FROM mine WHERE mine_guid NOT IN
(SELECT DISTINCT mine_guid FROM mine_party_appt WHERE mine_guid IS NOT NULL));` on prod is suggesting to me that some mines may have issues submitting reports.

Reply:
I don't disagree, but I think this would be better handled in it's own tech debt ticket.  partyRelationships are used in at least 13 different files and the clean up there will likely be somewhat involved.  Goes a bit out of the bounds of the camp site rule I think.

Reply:
Yeah mostly writing that down so that if such a bug surfaces, we'll be expecting it. I'm fine with it being out of scope.

Reply:
I'll write up a ticket for it as well

Reply:
I didn't get the excuse I was looking for to make a new redux slice, but hopefully that format is better at distinguishing "not loaded" and "no data"! And now that I look at it again, I don't think it's blocking loading, so it probably amounts to nothing. 👍